src/computation/logic/lague-dhondt.ts

```diff
@@ -139,7 +136,7 @@ export function lagueDhont(payload: ComputationPayload): LagueDhontResult {
     const districtResultArray = dictionaryToArray(districtResults);
     const partyResultArray = dictionaryToArray(partyResults);
 
-    const useAdjustedQuotients = payload.parameters.electionYear >= 2005;
+    const useAdjustedQuotients = payload.parameters.electionYear >= 2005 && isQuotientAlgorithm(payload.algorithm);
```

Comment:
2005 is a magic number.

Reply:
Do you have any suggestions for how to fix this? Hard to find a nice descriptive name for a variable or function.

Reply:
Cleaned up quite a few conditionals.